docs/edgeql/ddl/functions.rst

```diff
@@ -45,6 +45,7 @@ CREATE FUNCTION
 
     # where <subcommand> is one of
 
+      SET session_only := {true | false}
```

Comment:
Move this to below the `FROM` clause to group with `SET ANNOTATION`.

Reply:
OK, I grouped the "SET" commands together.